Automattic/wordpress-importer
# Source the WP_Import class regardless of request context

This change short-circuits the instantiation instead of the declaration of the `WP_Import` class (& other backing resources).

This is instead of preventing the file from being sourced altogether when the `WP_LOAD_IMPORTERS` constant is not defined.

As is, it's difficult (if not impossible) to modify that constant in a plugin prior to its evaluation here.

---

I inspected what runs when the file is sourced and there doesn't seem to be anything that's resource intensive or that would introduce side-effects.

---

This is the same commit as https://github.com/WordPress/wordpress-importer/pull/45 so we can get it into our `develop` branch.

### To Test

* Run this branch
* All unit test should still pass
* The WordPress import UI in wp-admin should "still work"™️
* No additional load should be discernible on regular or admin page loads (including the import pages)
